ts/Extensions/Boost/BoostSeries.ts

```diff
@@ -874,17 +874,29 @@ function getPoint(
         return boostPoint as BoostPointComposition;
     }
 
-    const xData = (
+    const isScatter = series.type === 'scatter',
```

Comment:
By using the `is` function, we ensure inherited series types are covered.

```suggestion
    const isScatter = series.is('scatter'),
```